### PR TITLE
add-jwtToken-cli-args

### PR DESCRIPTION
**Pull request by nehaabrol87**
@jonathanbarton  Added the jwtToken as part of query param.

### CODE REVIEW

**Comment by jonathanbarton** [comment]
lets remove user_id. As it is right now in the python script, its a required arg. I think we both came to the conclusion that user_id isnt needed for a POST telem..

**Comment by nehaabrol87** [comment]
aah yes let me do that.

File: device/rockblock.py [hunk 1]
```diff
@@ -20,11 +19,9 @@ class RockBlock():
 		iridium_cep: 4.0
 		data: 6c617469747564653d34322e33333437313035266c6f6e6769747564653d2d37322e3638303731393833333326616c7469747564653d303030373826736174656c6c697465733d3130266669785f7175616c6974793d3226536f756e643d313337264261726f6d657465723d313030312e34362654656d70657261747572653d34312e3030
 	"""
-	def __init__(self, url, flightname, device_id=None, user_id=None):
-		jwt_token = os.environ.get('GM_JWT_SECRET')
-		self.auth = JWTAuth(jwt_token)
+	def __init__(self, url, flightname, device_id=None, user_id=None, jwt_token=None):
 		self.user_id = user_id
-		self.url = url + '/flight/' + flightname + '/telemetry'
+		self.url = url + '/flight/' + flightname + '/telemetry?authorization=' + jwt_token
```
Comment (jonathanbarton) [inline device/rockblock.py]: Im sorry I didnt do this earlier:
`'%s/flight/%s/telemetry?authorization=%s'.format(url, flightname, jwt_token)`

Comment (jonathanbarton) [reply device/rockblock.py]: (python template strings)

Comment (nehaabrol87) [reply device/rockblock.py]: never too late.Let me add that.Looks much cleaner.

**Comment by nehaabrol87** [comment]
@jonathanbarton  Changed.Apparently {} works vs %s
Looks like %s  is for older versions.

**Comment by jonathanbarton** [comment]
LGTM

**Pull request merged**